cadquery/occ_impl/jupyter_tools.py

```diff
@@ -98,6 +98,6 @@ def x3d_display(shape,
         vec = quat*(vec) + c.wrapped
 
         # return boilerplate + Scene
-        return add_x3d_boilerplate(ElementTree.tostring(scene_tag),
+        return add_x3d_boilerplate(str(ElementTree.tostring(scene_tag).decode('utf-8')),
```

Comment:
str call seems not to be needed for python2.  _repr_html_ can return unicode on py2

Reply:
good point ;), removed in ad6d0b0b9356efbfe6c3d95d91315e02995f6858